app/assets/stylesheets/pages/articles/article-detail/_share.scss

```diff
@@ -0,0 +1,138 @@
+.share{
+	margin-top: 42px;
+
+	&__txt{
+		color: $textColorSecondary;
+		text-transform: capitalize;
+	}
+
+	&__sosmed-wrapper{
+		display: inline-block;
+
+		@media only screen and (max-width: 375px){
+			display: block;
+		}
+	}
+
+	&__sosmed-link{
+		display: inline-block;
+		background-color: $colorPrimary;
+		padding: 4px;
+		border-radius: 8px;
+		margin: 3px 10px 0 0;
+		transition: all .3s ease-in-out;
+
+		@media only screen and (max-width: 440px){
+			margin: 3px 5px 0 0;
+		}
+	}
+
+	&__sosmed-icons{
+		&--facebook{
+			background-image: image-url("icons/facebook.svg");
+			height: 35px;
+			width: 35px;
+			background-size: contain;
+			transform: scale(.9);
+			transition: all .3s ease-in-out;
+
+			&:hover{
+				transform: scale(1);
+			}
+
+			@media only screen and (max-width: 440px){
+				height: 30px;
+				width: 30px;
+			}
+		}
+
+		&--whatsapp{
+			background-image: image-url("icons/whatsapp.svg");
+			height: 35px;
+			width: 35px;
+			background-size: contain;
+			transform: scale(.9);
+			transition: all .3s ease-in-out;
+
+			&:hover{
+				transform: scale(1);
+			}
+
+			@media only screen and (max-width: 440px){
+				height: 30px;
+				width: 30px;
+			}
+		}
+
+		&--line{
+			background-image: image-url("icons/line.svg");
+			height: 35px;
+			width: 35px;
+			background-size: contain;
+			transform: scale(.9);
+			transition: all .3s ease-in-out;
+
+			&:hover{
```

Comment:
give the hover on the sosmed-icons class, because the user not always hovering the icon

Reply:
please check, is that like that?